src/functions/bitcoin/bitcoin-functions.ts

```diff
@@ -46,6 +47,17 @@ export function getFeeAmount(bitcoinAmount: number, feeBasisPoints: number): num
   return new Decimal(bitcoinAmount).times(feePercentage).trunc().toNumber();
 }
 
+export function removeDustOutputs(
+  outputs: { address: string; amount: bigint }[],
+  dustLimit = DUST_LIMIT
+): void {
+  for (let i = outputs.length - 1; i >= 0; i--) {
+    if (outputs[i].amount < dustLimit) {
+      outputs.splice(i, 1);
+    }
```

Comment:
Don't you want to do this with an array filter?

Reply:
I could have done that, but that doesn't affect the original array, but it creates a new one, and I did not want to use a new variable.